content/download/mirrors.adoc

```diff
@@ -9,13 +9,13 @@
 ---
 ## Releases
 
-Currently releases are mirrored to the locations documented on the link:https://updates.jenkins.io/download/war/latest/jenkins.war?mirrorstats[mirror status page].
+Currently releases are mirrored to the locations documented on the link:https://get.jenkins.io/index.html?mirrorstats[mirror status page].
 
 ### Running a mirror
 
 More mirrors are certainly welcome.
-Jenkins is mirrored by the link:https://osuosl.org/services/hosting/details[Oregon State University Open Source Lab (OSUOSL)] and by servers hosted by donors in the United States, Europe, China, and Japan.
+Jenkins is mirrored by the link:https://osuosl.org/services/hosting/details[Oregon State University Open Source Lab (OSUOSL)] and by servers hosted by donors in the United States, Europe, China, Singapore and Japan.
 Worldwide mirrors reduce the load on individual servers, spread the bandwidth costs among multiple servers, and improve download performance for users.
 
-If you're interested in running a public mirror, drop an email to `jenkins-infra@googlegroups.com` and we can discuss space and bandwidth requirements.
-Mirrors on the Indian subcontinent, in South America, and in Africa would be great additions to the Jenkins mirrors.
+If you're interested in running a public mirror, open an issue in link:https://github.com/jenkins-infra/helpdesk[] and we can discuss space and bandwidth requirements.
```

Comment:
Rather than show it as a link, I prefer text that describes the purpose of the link.  Optional.
```suggestion
If you're interested in running a public mirror, open an issue in the link:https://github.com/jenkins-infra/helpdesk[Jenkins infra help desk] and we can discuss space and bandwidth requirements.
```

Reply:
I'm usually not a fan of link titles but using one maps to the site de-factor usage so LGTM